notebooks/data-sources/github/thoth_PR_data_extraction.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # List of repos we will be extracting PR data
</pre><p>Can you add a comment here linking the notebook to show how we came up with the list?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>